docs/UserGuide.md

```diff
@@ -154,13 +156,13 @@ Example: `add` Watch webcast <i>on</i> sat <i>from</i> 4.45pm <i>to</i> 7pm
 - `add` TASK <i>from</i> DATE <i>to</i> DATE <i>from</i> TIME <i>to</i> TIME [<i>#</i>TAG] [<i>p/</i>PRIORITY]  [<i>r/</i>RECURRING] <br>
 Example: `add` Holiday in San Francisco at Ocean Beach by the sea <i>from</i> may <i>to</i> aug <i>from</i> 9am <i>to</i> 11pm <i>#</i>holiday <i>#</i>leisure <i>p/</i>3 <i>r/</i>monthly
 
```

Comment:
![image](https://cloud.githubusercontent.com/assets/13817632/20036880/d5d34760-a44f-11e6-9d35-9585e56eb095.png)
Format and example not consistent